cli/prompt_secret.ts

```diff
@@ -0,0 +1,57 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+
+const input = Deno.stdin;
+const output = Deno.stdout;
+const encoder = new TextEncoder();
+const decoder = new TextDecoder();
+const LF = "\n".charCodeAt(0);
+const CR = "\r".charCodeAt(0);
+
+/**
+ * Shows the given message and waits for the user's input. Returns the user's input as string.
+ * This is similar to `prompt()` but it print user's input as `*` to prevent password from being shown.
+ * Use an empty `mask` if you don't want to show any character.
+ */
+export function promptSecret(
+  message = "Password ",
```

Comment:
```suggestion
  message = "Secret ",
```
A secret may not always be a password.